psalm.xml

```diff
@@ -1,25 +1,25 @@
 <?xml version="1.0"?>
 <psalm
-    errorLevel="2"
```

Comment:
We are at error level 2 which should already throw errors if we have deprecations in use:

https://psalm.dev/docs/running_psalm/error_levels/